string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,6 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  let reversedStr = '';
+  for (let i = word.length - 1; i >= 0; i--) { reversedStr += word.charAt(i); }
```

Comment:
You can use bracket notation on strings so instead of `word.charAt(i)` you can use `word[i]`